aws/DRVault-Single-Deployment.yaml

```diff
@@ -495,6 +511,17 @@ Parameters:
       - Accept
       - Decline
     Default: Decline
+  ReleaseVersion:
```

Comment:
Can we also fetch this from AWS like we do with the AMI ID ?

Reply:
Not sure I understand

Reply:
Impossible, because the user needs to receive the available versions before the CloudFormation execution starts